lib/views.js

```diff
@@ -0,0 +1,30 @@
+var optionDescriptions = require('../lib/optionDescriptions');
+
+this.Group = {
+	properties: {
+		'name': {
+			message: optionDescriptions.GROUP_NAME,
+			required: true
+		}
+	}
+}
+
+
+this.Login = {
+	properties: {
+        host: {
+            default: 'cloud.mongodb.com',
+            message: optionDescriptions.HOST,
+            required: true
+        },
+        user: {
+            message: optionDescriptions.USER,
+            required: true
+        },
+        apiKey: {
+            message: optionDescriptions.API_KEY_,
+            required: true,
+            hidden: true
+        }
+    }
+}
```

Comment:
I'm not sure about naming this Views.  I tend to think of Views as a presentation concept.  Maybe Forms? or Schema?
Also, convention is generally to return with module.exports.

Otherwise LGTM :+1: 
I'm going to merge and resolve the conflicts with my branch.